### PR TITLE
[HCAP-928] security hot spot

### PR DESCRIPTION
**Pull request by vyasworks**

### CODE REVIEW

File: server/routes/participant.js [hunk 1]
```diff
@@ -181,13 +181,19 @@ participantsRouter.post(
   keycloak.getUserInfoMiddleware(),
   multer({
```
Comment (fwpushan) [inline server/routes/participant.js:182]: possibly multer has input param `size: `

**Review by fwpushan — COMMENTED** [review]
LGTM. Just one minor remark.

File: server/routes/participant.js (hunk 1 above)
Comment (vyasworks) [reply server/routes/participant.js:182]: I tried `limits` but I could not get it to reject the file :thinking:

Comment (vyasworks) [reply server/routes/participant.js:182]: ```    limits: { fileSize: 6000 },```

**Review by fwkendall — APPROVED** [review]

**Review by fwpushan — APPROVED** [review]

**Pull request merged**